{{cookiecutter.project_slug}}/config/settings/base.py

```diff
@@ -187,6 +187,7 @@
                 "django.template.context_processors.static",
                 "django.template.context_processors.tz",
                 "django.contrib.messages.context_processors.messages",
+                "context_processors.settings_context",
```

Comment:
I think this should be located inside a package. It doesn't really belong in any app, so maybe it's time to create a `utils` package next to the `users` app?